include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -2029,23 +2098,24 @@ MGTwoLevelTransfer<dim, LinearAlgebra::distributed::Vector<Number>>::prolongate(
     this->vec_coarse_constraints.copy_locally_owned_data_from(src);
     this->vec_coarse_constraints.update_ghost_values();
 
-    for (const auto i : partitioner_coarse->locally_owned_range())
-      {
-        if (constraint_coarse.is_constrained(i) == false)
-          continue;
-
-        Number temp = constraint_coarse.is_inhomogeneously_constrained(i) ?
-                        constraint_coarse.get_inhomogeneity(i) :
-                        0.0;
-
-        const auto constraints = constraint_coarse.get_constraint_entries(i);
+    for (unsigned int i = 0; i < constraint_coarse_distribute_ptr.size() - 1;
+         ++i)
+      if ((constraint_coarse_distribute_ptr[i + 1] -
+           constraint_coarse_distribute_ptr[i]) == 0)
```

Comment:
I guess the compile will do it automatically, but still I would prefer
```suggestion
      if (constraint_coarse_distribute_ptr[i + 1] ==
           constraint_coarse_distribute_ptr[i])
```